### ISSUE TITLE
Firebase (firebase.com) with AngularFire cannot be installed

### ISSUE THREAD

**Issue by ix-xerri** [mgechev/angular2-seed#1156]
The first thing I ran after cloning this repo was `npm install angularfire2 firebase --save` to install AngularFire and start getting data from Firebase but it does not work. A lot of peerDepedencies errors.

**Comment by mgechev** [comment]
What errors do you see? It is quite possible that the problem is not related to the angular2-seed.

**Comment by ix-xerri** [comment]
A lot of peerDependencies error. This was the first thing I tied to do so
no other changes.

**Comment by mgechev** [comment]
Would you share the errors here?

**Comment by ix-xerri** [comment]
`npm WARN peerDependencies The peer dependency @angular/core@2.0.0-rc.2 included from angularfire2 will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN peerDependencies The peer dependency @angular/platform-browser@2.0.0-rc.2 included from angularfire2 will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN peerDependencies The peer dependency @angular/common@2.0.0-rc.2 included from angularfire2 will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN peerDependencies The peer dependency @angular/compiler@2.0.0-rc.2 included from angularfire2 will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN peerDependencies The peer dependency @angular/platform-browser-dynamic@2.0.0-rc.2 included from angularfire2 will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm ERR! Darwin 15.6.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "angularfire2" "firebase" "--save"
npm ERR! node v4.4.5
npm ERR! npm  v2.15.5
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package @angular/common@2.0.0-rc.4 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer @angular/forms@0.2.0 wants @angular/common@^2.0.0-rc.4
npm ERR! peerinvalid Peer @angular/platform-browser@2.0.0-rc.4 wants @angular/common@^2.0.0-rc.4
npm ERR! peerinvalid Peer @angular/platform-browser-dynamic@2.0.0-rc.4 wants @angular/common@^2.0.0-rc.4
npm ERR! peerinvalid Peer @angular/router@3.0.0-beta.2 wants @angular/common@^2.0.0-rc.4
npm ERR! peerinvalid Peer angularfire2@2.0.0-beta.2 wants @angular/common@2.0.0-rc.2
`

**Comment by mgechev** [comment]
The issue is not related to the seed https://github.com/angular/angularfire2/issues/388. You can also follow https://github.com/angular/angularfire2/pull/387.

**Comment by ix-xerri** [comment]
I am monitoring for any changes. Do you think I can set it to install the latest of everything and ignore dependencies? How can I do that?

**Comment by ix-xerri** [comment]
just bootstrapped https://github.com/jeffbcross/angularfire2-seed which is based on this repo

**Comment by mgechev** [comment]
Yes, ignoring peerDependency conflicts should work sine angularfire2 is compatible with rc.4 as well.

**Comment by ix-xerri** [comment]
I am back to using this branch. I upgraded nam and node to make sure I have the latest and started from scratch. I have also found you article about [AngularFire2](https://github.com/mgechev/angular2-seed/wiki/Integration-with-AngularFire2). Running the first command generated the following errors. Any recommendation?

`npm WARN angularfire2@2.0.0-beta.2 requires a peer of @angular/core@2.0.0-rc.2 but none was installed.
npm WARN angularfire2@2.0.0-beta.2 requires a peer of @angular/platform-browser@2.0.0-rc.2 but none was installed.
npm WARN angularfire2@2.0.0-beta.2 requires a peer of @angular/common@2.0.0-rc.2 but none was installed.
npm WARN angularfire2@2.0.0-beta.2 requires a peer of @angular/compiler@2.0.0-rc.2 but none was installed.
npm WARN angularfire2@2.0.0-beta.2 requires a peer of @angular/platform-browser-dynamic@2.0.0-rc.2 but none was installed.`
